core/src/main/scala/tofu/lift/IsoK.scala

```diff
@@ -30,4 +30,11 @@ object IsoK {
     def to[A](fa: F[A]): F[A]   = fa
     def from[A](ga: F[A]): F[A] = ga
   }
+
+  def byFunK[F[_], G[_]](fg: F ~> G, gf: G ~> F): IsoK[F, G] =
```

Comment:
I would like to have `(fg: F ~> G)(gf: G ~> F)` to make it consistent with methods from `tofu.syntax.lift._`